x-pack/plugins/cloud_security_posture/common/schemas/csp_configuration.ts

```diff
@@ -6,7 +6,7 @@
  */
 import { schema as rt, TypeOf } from '@kbn/config-schema';
 
-// cspRulesConfigSchema have to be correspond to DataYaml struct in https://github.com/elastic/cloudbeat/blob/main/config/config.go#L44-L50
+// cspRulesConfigSchema have to be corresponded to DataYaml struct in https://github.com/elastic/cloudbeat/blob/main/config/config.go#L44-L50
```

Comment:
nit: `cspRulesConfigSchema has to match the 'DataYaml' struct in`

Reply:
fixed